mlrun/model_monitoring/controller.py

```diff
@@ -71,14 +71,23 @@ def _get_last_analyzed(self) -> Optional[int]:
         except HttpResponseError as err:
             logger.warn(
                 "Failed to get the last analyzed time for this endpoint and application, "
-                "as this is probably the first time this application is running. "
-                "Using the first request time instead.",
+                "as this is probably the first time this application is running. ",
+                "Using the latest between first request time or last update time instead.",
```

Comment:
Log the time used as "last_analyzed" and where it comes from (stop - 1d / first_request).